dojo/tools/nexpose/parser.py

```diff
@@ -252,5 +252,52 @@ def get_items(self, tree, vulns, test):
                     if 'port' in service:
                         endpoint.port = int(service['port'])
                     find.unsaved_endpoints.append(endpoint)
+                    find.unsaved_tags = list()
+                    find.unsaved_tags = vuln['tags']
+
+        # manage findings by node only
+        for vuln in host['vulns']:
+            dupe_key = vuln['severity'] + vuln['name']
+
+            if dupe_key in dupes:
+                find = dupes[dupe_key]
+                dupe_text = html2text.html2text(vuln['pluginOutput'])
+                if dupe_text not in find.description:
+                    find.description += "\n\n" + dupe_text
+            else:
+                find = Finding(title=vuln['name'],
+                                description=html2text.html2text(
+                    vuln['desc'].strip()) + "\n\n" + html2text.html2text(vuln['pluginOutput'].strip()),
+                    severity=vuln['severity'],
+                    numerical_severity=Finding.get_numerical_severity(vuln['severity']),
+                    mitigation=html2text.html2text(vuln['resolution']),
+                    impact=vuln['vector'],
+                    test=test,
+                    active=False,
+                    verified=False,
+                    false_p=False,
+                    duplicate=False,
+                    out_of_scope=False,
+                    mitigated=None,
+                    dynamic_finding=True)
+                # build references
+                refs = ''
+                for ref in vuln['refs']:
+                    if ref.startswith('CA'):
+                        ref = f" * [{vuln['refs'][ref]}](https://www.cert.org/advisories/{vuln['refs'][ref]}.html)"
+                    elif ref.startswith('CVE'):
+                        ref = f" * [{vuln['refs'][ref]}](https://cve.mitre.org/cgi-bin/cvename.cgi?name={vuln['refs'][ref]})"
```

Comment:
Same like https://github.com/DefectDojo/django-DefectDojo/pull/3872/files#r576756981

Reply:
not my original code but fixed in 4 places. :)